Palindrome.java

```diff
@@ -0,0 +1,17 @@
+class PalindromeExample{  
+ public static void main(String args[]){  
+  int r,sum=0,temp;    
+  int n=454;//It is the number variable to be checked for palindrome  
```

Comment:
rename n to more descriptive variable : example numberToCheck or numberToCheckForPalindrome (might be too long )